explorer/client/package.json

```diff
@@ -42,7 +42,6 @@
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-ga4": "^1.4.1",
-        "react-json-view": "^1.21.3",
```

Comment:
why remove the react-json-view how are we dealing with struct/ JSON object view ?

Reply:
The Figma Design requires that the JSON data is displayed in the same way as the Modules. Hence, the same ModulesWrapper component is used for both.